Amazon.QLDB.Driver.Tests/TransactionTests.cs

```diff
@@ -66,7 +66,7 @@ public void TestDisposeWhenExceptionIsProperlyIgnored()
         public void TestCommitCallsCommitTransaction()
         {
             mockSession.Setup(m => m.CommitTransaction(It.IsAny<string>(), It.IsAny<MemoryStream>()))
-                .Returns(GetTransactionResult(TxnId));
+                .Returns(AsyncTransactionTests.GetTransactionResult(TxnId).GetAwaiter().GetResult());
```

Comment:
Can we define a quick one line function in this file that does this rather than repeat this async code throughout these non-async tests?

Reply:
done